vulnerabilities/scraper/safety_db.py

```diff
@@ -0,0 +1,60 @@
+# Data Imported from https://github.com/pyupio/safety-db
+import json
+from urllib.error import HTTPError
+from urllib.request import urlopen
+from dephell_specifier import RangeSpecifier
+
+URL = 'https://raw.githubusercontent.com/pyupio/safety-db/master/data/insecure_full.json'
+
+
+def get_data_file():
+    with urlopen(URL) as file:
+        json_file = json.load(file)
+    return json_file
+
+
+def get_all_versions_of_package(package_name):
+    url = "https://pypi.org/pypi/{}/json".format(package_name)
+    try:
+        with urlopen(url) as response:
+            json_file = json.load(response)
+    except HTTPError:  # PyPi does not have data about this package, we skip these
+        return set()
+    for release in json_file['releases']:
+        yield release
+
+
+def import_vulnerabilities():
+    vulnerability_package_dicts = []
+    data = get_data_file()
+    cves = set()
+    for package in data:
+        package_name = package
```

Comment:
Why not call the variable `package_name` right away (in the for loop) if you think this is more descriptive?